src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -694,6 +694,31 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 		const struct xccdf_version_info *xccdf_version = xccdf_benchmark_get_schema_version(benchmark);
 		const char *xccdf_version_name = xccdf_version_info_get_version(xccdf_version);
 
+		if (NULL != profile_description) {
+			unsigned int newLines = 0;
+			const char *c = profile_description;			
+			while (*c != '\0') {
+				if (*c == '\n')
+					++newLines;
+				++c;
+			}
+			
+			if (newLines > 0) {
+				unsigned int description_length = c - profile_description;
+				const char filler[] = "#                       ";
```

Comment:
why `filler[]` and not `filler*`?

Reply:
So I can use sizeof
  const char *str1 = "Hello World";
  const char str2[] = "Hello World";
  std::cout << sizeof str1 << std::endl;
  std::cout << sizeof str2 << std::endl;
---------------
8
12